<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleSiteEditorUi</class>
 <widget class="QWidget" name="SimpleSiteEditorUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>299</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QListWidget" name="pSimplePropertiesListWidget">
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Common</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Message View</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="msgSysRadioButton">
          <property name="toolTip">
           <string>Plain text baloon popup in System Tray</string>
          </property>
          <property name="text">
           <string>Text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="msgSysExRadioButton">
          <property name="toolTip">
           <string>Interactive EWA's widget</string>
          </property>
          <property name="text">
           <string>Widget</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Caption:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="titleLineEdit">
         <property name="text">
          <string>EWA [%url%]</string>
         </property>
         <property name="maxLength">
          <number>64</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Duration:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="durationSpinBox">
         <property name="toolTip">
          <string>Message duration time 
( 10-600 seconds )</string>
         </property>
         <property name="suffix">
          <string> s</string>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>600</number>
         </property>
         <property name="value">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Autocheck every:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="timeValueSpinBox">
           <property name="toolTip">
            <string>Download period ( units count )</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="timeUnitsComboBox">
           <property name="toolTip">
            <string>Download period ( units )</string>
           </property>
           <item>
            <property name="text">
             <string>seconds</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>minutes</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>hours</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="selectTargetButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Select target</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>:/images/select_target.png</normaloff>:/images/select_target.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="changesCheckBox">
         <property name="text">
          <string>Monitor changes:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="EWASiteWidgetColorManager" name="pXMsgColorManager" native="true"/>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>121</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EWASiteWidgetColorManager</class>
   <extends>QWidget</extends>
   <header>ewa_sitewidgetcolormanager.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>msgSysRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>pXMsgColorManager</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>changesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>selectTargetButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>475</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
